central/role/resources/list.go

```diff
@@ -32,6 +32,7 @@ var (
 	BackupPlugins                    = newResourceMetadata("BackupPlugins", permissions.GlobalScope)
 	CVE                              = newResourceMetadata("CVE", permissions.NamespaceScope)
 	Cluster                          = newResourceMetadata("Cluster", permissions.ClusterScope)
+	ClusterCVE                       = newResourceMetadata("ClusterCVE", permissions.ClusterScope)
```

Comment:
I am discussing this with SAC team. SAC for some resources is derived from others. Listing resource here indicates that explicit permission to that resource is required, which is not always true.
But without adding the resource here, the generation fails.